vladiate/vlad.py

```diff
@@ -85,6 +86,7 @@ def validate(self):
             return False
 
         for line, row in enumerate(reader):
+            self.line_count = line
```

Comment:
This will set `self.line_count` multiple times until the loop has finished, and since `enumerate` produces a zero-indexed list, this causes an off-by-one error where `self.line_count` is one less than the actual total number of lines.

Instead, you might want to do `self.line_count += 1`.